README.md

```diff
@@ -101,7 +101,19 @@ See [Security filters for trimming Azure Search results using Active Directory i
 
 ## Running the DotNetHowToAutocomplete sample
 
+This sample demonstrates how to create an index and a synonym-map, both encrypted using a customer-managed key in Azure Key Vault. To learn more, see [Azure Search encryption using customer-managed keys in Azure Key Vault](https://docs.microsoft.com/azure/search/search-security-manage-encryption-keys).
+
+To run this sample, in addition to an Azure Search service, you would have to have access to an Azure Key Vault account, and optionally also to an Azure Active Directory application. 
+You would need to obtain the key-identifier of some Azure Key Vault key, that has at least 'get', 'wrap' and 'unwrap' capabilities. An Azure Key Vault key identifier has the following format: 'https://<key-vault-name>.vault.azure.net/keys/<key-name>/<key-version>'. 
+
+In order for your Azure Search service to be able to use your Azure Key Vault key, you must grant your search service certain Key Vault access permissions, specifically 'get', 'wrap' and 'unwrap'.
+If your Azure Search service has a managed service identity (see [Create a service identity](https://docs.microsoft.com/en-us/azure/search/search-security-manage-encryption-keys#3---create-a-service-identity)), you can grant these permissions from your key vault directly to your search service. If MSI is not enabled for your search service, and you don't wish to enable it, you must create (or use existing) Azure Active Directory (AAD) application, grant that application the required access permissions, and share the credentials of this AAD application with your Azure Search service. Learn more about this option here - [Use an externally managed Azure Active Directory application](https://docs.microsoft.com/en-us/azure/search/search-security-manage-encryption-keys#aad-app).
+
+After obtaining the required resources described about, follow this steps to run this sample:
+
 * Open the DotNetHowToAutocomplete.sln project in Visual Studio
+* Update the appsettings.json with your Azure Search service name and api-keys, and your Azure Key Vault key identifier

```

Comment:
All of the above steps should be documented somewhere else, right? Like in a conceptual article? Maybe you could just link to that.

Reply:
I do link to that in the very first sentence (line#104)…

Reply:
Ah, I missed that. Maybe not worth repeating the details here in that case. Up to you.